addons/CLib/InitFunctions/initCalls.sqf

```diff
@@ -70,8 +70,12 @@ if (_recompile == 3) then {
         private ["_test", "_test2"];
         _test = BIS_functions_mainscope setPos (getPos BIS_functions_mainscope);
         _test2 = BIS_functions_mainscope playMove "";
-        if (isNil "_test") then {_test = false};
-        if (isNil "_test2") then {_test2 = false};
+        if (isNil "_test") then {
```

Comment:
i think if then {} simple expressions should be allowed in 1 line. that can improve Visual Aspect of the code in some cases. but maybe we can build in a max char count?